nix/installer.nix

```diff
@@ -11,13 +11,61 @@
   }).latestCompatibleLinuxPackages;
   boot.zfs.removeLinuxDRM = lib.mkDefault pkgs.hostPlatform.isAarch64;
 
-  documentation.enable = false;
+  documentation.enable = lib.mkForce false;
+
+  # reduce closure size through package set crafting
+  # where there's no otherwise globally effective
+  # config setting available
+  # TODO: some are candidates for a long-term upstream solution
+  nixpkgs.overlays = [
+    (final: prev: {
+      # save ~12MB by not bundling manpages
+      coreutils-full = prev.coreutils;
+      # save ~20MB by making them minimal
+      util-linux = prev.util-linux.override {
+        nlsSupport = false;
+        ncursesSupport = false;
+        systemdSupport = false;
```

Comment:
Don't we depend on systemd anyway?

Reply:
True, the final NotOS implementation also does not (any more) depend on `nix/installer.nix` (which had been the hidden motive).

I'll remove that line. Thanks!